testem.json

```diff
@@ -5,6 +5,7 @@
     "lib/loader/loader.js",
     "tests/all.js"
   ],
+  "before_tests": "npm run prepublish",
```

Comment:
We may want to consider a different script, as I believe prepublish is deprecated in npm 4. http://blog.lholmquist.org/npm-prepublish-changes/